tools/gluten-te/ubuntu/dockerfile-build

```diff
@@ -93,6 +93,7 @@ RUN EXTRA_MAVEN_OPTIONS=$(cat ~/.gluten-mvn-options) \
     DEPS_INSTALL_SCRIPT=$(cat ~/.gluten-deps-install-script) \
     && cd /opt/gluten \
     && bash -c "$DEPS_INSTALL_SCRIPT" \
-    && bash -c "mvn clean install $GLUTEN_MAVEN_OPTIONS $EXTRA_MAVEN_OPTIONS"
+    && bash -c "mvn clean install $GLUTEN_MAVEN_OPTIONS $EXTRA_MAVEN_OPTIONS" \
+    && bash -c "mv ep/build-velox/build/velox_ep /opt/velox"
```

Comment:
this is to save velox build to /opt

Reply:
Minor: This would make libvelox.so unable to locate symbol source when debug build is enabled. Is there any reason to do it?

Reply:
I'm preparing the pre-built docker image for GHA. In GHA The velox home is set to /opt/velox. by save the pre-built velox to `/opt/velox `can help to reuse the velox build before rebase happen. We could also try to copy the velox folder to /opt/ but it will occupy too many disk spaces 

https://github.com/oap-project/gluten/blob/main/.github/workflows/velox_be.yml#L59

A proper way is to create separate docker files for dev, CI image purpose - we could follow up on this